internal/cmd/ansible-operator/run/cmd.go

```diff
@@ -330,6 +332,20 @@ func getAnsibleDebugLog() bool {
 	return val
 }
 
+// getAnsibleDebugLog return the integer value of the log level set in the flag
+func getAnsibleLogLevel(f *flags.Flags) events.LogLevel {
```

Comment:
I might name this something other than ansible log level, as that sounds like it applies to the ANSIBLE_VERBOSITY value, whereas this is just determining what classes of things get logged rather than what verbosity. I might just call it `getAnsibleEventsToLog` or something that makes it a little less muddy since it's so confusing already

Reply:
@fabianvf I ran a playbook with a single `hello world` debug task, it only got printed once when the flag is set to `Everything`, so I don't think the expected output gets logged twice.